core/src/main/java/com/scalar/db/api/DeleteBuilder.java

```diff
@@ -90,10 +99,31 @@ public Buildable consistency(com.scalar.db.api.Consistency consistency) {
       return this;
     }
 
+    @Override
+    public Buildable attribute(String name, String value) {
```

Comment:
It's again a super minor comment, but the definition order of the variables (`consistency`, `condition`, and `attributes`) and the method definition order can be consistent to make it slightly better.
It's most probably my preference, so I don't argue about it.

Reply:
Let me make them consistent. Thanks.

Reply:
Ah, I thought I would do that this time, but it looks like it’s very time-consuming… Actually, the original method definition order is already inconsistent. Let’s tackle it when we have more time. 🙇